dispatches/case_studies/renewables_case/tests/test_surrogate_opt.py

```diff
@@ -0,0 +1,41 @@
+#################################################################################
+# DISPATCHES was produced under the DOE Design Integration and Synthesis Platform
+# to Advance Tightly Coupled Hybrid Energy Systems program (DISPATCHES), and is
+# copyright (c) 2020-2023 by the software owners: The Regents of the University
+# of California, through Lawrence Berkeley National Laboratory, National
+# Technology & Engineering Solutions of Sandia, LLC, Alliance for Sustainable
+# Energy, LLC, Battelle Energy Alliance, LLC, University of Notre Dame du Lac, et
+# al. All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. Both files are also available online at the URL:
+# "https://github.com/gmlc-dispatches/dispatches".
+#################################################################################
+import pytest
+
```

Comment:
@dguittet you can add the following line to skip the entire test file when `tensorflow` & co. are not installed:

```suggestion

pytest.importorskip("tensorflow", reason="optional dependencies for surrogate modeling not available")
```